features/pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright 2016-2020 The OpenZipkin Authors
+    Copyright 2016-2024 The OpenZipkin Authors
```

Comment:
unrelated.. at some point this could be refactored to use the short license headers

Reply:
> unrelated.. at some point this could be refactored to use the short license headers

Yep, will do shortly with Brave 6 / Zipkin 3 update, thanks!